scripts/anonymizer.py

```diff
@@ -0,0 +1,189 @@
+"""Module that handles the full Anonymization pipeline
+"""
+import sys
+import xml.etree.ElementTree as ET
+
+def rewriteFakeQueries(fakeValues, path):
```

Comment:
I think that the API currently lacks context. Maybe add a short flow description in the overview/main somewhere to explain why you need these specific functions.